tests/sqlparser_common.rs

```diff
@@ -272,20 +275,39 @@ fn parse_insert_default_values() {
         "INSERT INTO test_table DEFAULT VALUES (some_column)";
     assert_eq!(
         ParserError::ParserError("Expected: end of statement, found: (".to_string()),
-        parse_sql_statements(insert_with_default_values_and_hive_after_columns).unwrap_err()
+        all_dialects_requiring_semicolon_statement_delimiter()
```

Comment:
this query doesn't seem to have multiple statements. Why does the test need to be changed?

Reply:
Turning on the option to make semicolons optional results in this test failures:

```
thread 'parse_insert_default_values' panicked at src/test_utils.rs:101:21:
assertion `left == right` failed: Parse results with PostgreSqlDialect are different from MsSqlDialect
  left: Err(ParserError("Expected: end of statement, found: ("))
 right: Err(ParserError("Expected: SELECT, VALUES, or a subquery in the query body, found: some_column"))
```

Since that's a parser error in both cases (but different message) I split the assertion out into two -- one for semicolons required that asserts the former error, and one for semicolons optional that asserts the other error.